pipeline/scripts/3-averageTechReplicates.R

```diff
@@ -122,3 +124,61 @@ repl.pattern.filtered <- retVal$pattern
 save(repl.pattern.filtered, file=paste(outdir, "repl.pattern.negative.RData", sep="/"))
 write.table(remove_neg, file=paste(outdir, "miss_infusions_neg.txt", sep="/"), row.names=FALSE, col.names=FALSE ,sep= "\t")
 
+# New: generate TIC plots
+run_name <- basename(outdir)
+TIC_input_dir <- paste(outdir, "2-pklist", sep = "/")
+
+# get replication pattern
+load(paste(outdir, "logs", "init.RData", sep="/"))
+# get misinjections
+bad_pos <- read.table(paste(outdir, "miss_infusions_pos.txt", sep="/"))
+bad_neg <- read.table(paste(outdir, "miss_infusions_neg.txt", sep="/"))
+
+# get all txt files
+TIC.files = list.files(TIC_input_dir, full.names=TRUE, pattern="*TIC.txt")
```

Comment:
Be consistent with the style for variable names. I would recommend to stick to the underscore as separator for words as in the other variable names. The `.` can be confused with setting a variable in an object or calling a method of a class. Besides that, try to avoid capitals in your variable names since in most languages this indicates a global variable or a constant.

Reply:
Underscore used as separator in variable names. Capital TIC replaced with tic for variable names.